Resources/Prototypes/Entities/Objects/Decoration/posters.yml

```
@@ -0,0 +1,1148 @@
+- type: entity
+  parent: BaseItem
+  id: FoldedPosterBase
+  name: rolled poster
+  description: Make the propaganda work on you today!
+  abstract: true
+  components:
+  - type: Sprite
+    sprite: Structures/Wallmounts/posters.rsi
+    state: rolled_legit
+  - type: Item
+    size: Small
+  - type: StaticPrice
+    price: 100
+  - type: Tag
+    tags:
+    - Poster
+
```

```suggestion
    tags:
    - Poster
    - Trash
    - Recyclable

```

Makes folded posters recyclable.

---

They can make them recyclable, but no sure why we need to mark them as trash.